packages/launchpad/src/default-wired-launchpad.tsx

```diff
@@ -1,15 +1,20 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
 import {
 	LaunchpadNavigator,
+	Site,
+	SiteSelect,
```

Comment:
Nit: should this not be a type import?
```suggestion
	type SiteSelect,
```